pkg/apis/build/v1alpha1/build.go

```diff
@@ -96,3 +96,7 @@ func (b *Build) BuildEnvVars() []corev1.EnvVar {
 func (b *Build) ImagePullSecretsVolume() corev1.Volume {
 	return b.Spec.Source.Source().ImagePullSecretsVolume()
 }
+
+func (b *Build) Rebasable() bool {
+	return b.Annotations[BuildReasonConfig] == BuildReasonStack
```

Comment:
Where do we set this Annotation?

Reply:
https://github.com/pivotal/kpack/pull/140/files#diff-93376212cc28d900cfd5f6ab34de23d0L26-R63